compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -893,19 +893,63 @@ lcmpHelper(TR::Node * node, TR::CodeGenerator * cg)
    return targetRegister;
    }
 
+static TR::Register * maxMinHelper(TR::Node *node, TR::CodeGenerator *cg, bool isMax)
```

Comment:
Could we please Doxygen document this function and it's limitations (only handles 32-bit and 64-bit min/max).

Reply:
d420fe4